user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_deployment.md

```diff
@@ -6,24 +6,29 @@ uid: I-DOCSIS_deployment
 
 To deploy the I-DOCSIS branch of the EPM Solution:
 
-1. Make sure the **latest DataMiner feature release** version is installed.
+1. Make sure the latest DataMiner feature release version is installed.
 
-1. Place the **EPM package** somewhere on the server (in a different folder than C:\Skyline DataMiner) and unzip it.
+1. Deploy the EPM package:
 
-1. In DataMiner Cube, go to the Protocols & Templates module and **upload the necessary connectors** (called "protocols" in Cube) from the EPM package. See [Uploading a Protocol.xml file](xref:Adding_a_protocol_or_protocol_version_to_your_DataMiner_System#uploading-a-protocolxml-file).
+   - If the DMA is connected to dataminer.services, you can [deploy the package](xref:Deploying_a_catalog_item) directly from the Catalog.
+   - Otherwise, place the latest EPM package on the DMA server (in a different folder than C:\Skyline DataMiner) and install the package.
```

Comment:
Updated first sentence. The I-DOCSIS package does not support upgrading form the taskbar

Reply:
@SkylineGBO, since you say the package does not support upgrading from the taskbar, how should users then install the upgrade package on the server? Right now, we include the link to upgrading using the Taskbar Utility, but perhaps we'll have to remove this then?

Reply:
Hi @MariekeGO, I already sync with @SkylineGBO.
Packages can be installed from the Taskbar:
https://docs.dataminer.services/release-notes/General/General_Feature_Release_10.2/General_Feature_Release_10.2.10.html?q=taskbar%20dmapp#dataminer-taskbar-utility-enhanced-installation-of-app-packages-id_33969